Moderation-Policy.md

```diff
@@ -31,34 +31,33 @@ Any alternative Moderation Policy used for a given repository must be included i
 
 ### Terms
 
-* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
-* *TSC* refers to the [Node.js Technical Steering Committee](https://github.com/nodejs/node#tsc-technical-steering-committee).
+* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation][] for more information.
+* *TSC* refers to the [Node.js Technical Steering Committee][].
+* *CommComm* refers to the [Node.js Community Committee][].
 * *Post* refers to the content and titles of any issue, pull request, comment or wiki page.
 * *Moderate* refers to the act of modifying the content and title of, or deleting, any Post for the purpose of correcting or addressing Code of Conduct violations.
 * *Remove* refers to the act of removing the configured write (commit) permissions for an individual Collaborator's GitHub account from *all* Node.js GitHub Organization repositories as well as removing the account from the Node.js GitHub Organization membership.
-* *Ban* refers to the act of blocking an individual GitHub account from any further participation in the Node.js GitHub Organization.
+* *Ban* refers to the act of blocking an individual GitHub account from any further participation in the Node.js GitHub Organization. Bans may be *temporary* (24 hours) or *indefinite*.
 * *Requester* refers to an individual requesting Moderation on a Post.
 
 ### Grounds for Moderation
 
 Any Post considered to be in violation of the Node.js [Code of Conduct][] is subject to Moderation.
 
-The TSC is solely responsible for deciding what constitutes inappropriate behavior that may be subject to Moderation (see: [Escalation to the TSC](#escalation-to-the-tsc)).
-
 ### Requesting Moderation
 
 Anyone may request Moderation of a Post. Requesting Moderation of a Post can be accomplished in one of four ways:
 
 * Via the [report@nodejs.org](mailto:report@nodejs.org) email address,
-* Via private email to individual TSC members,
+* Via private email to individual Moderation Team members,
```

Comment:
for people outside of the loop, might it make sense to (eventually) link the moderation team member list here?